.github/benchmark-files/benchmark.ini

```diff
@@ -0,0 +1,11 @@
+max_execution_time=0
+error_reporting=0
+display_errors=off
+log_errors=off
+opcache.enable=1
```

Comment:
We enable the opcache both for cli and cgi modes, disable the jit compiler and disable opcache optimization (so that the case where Xdebug is not loaded matches the case where it is loaded as Xdebug disables this optimization) We also set opcache to use a file cache so that it can be shared between the two runs that we do of the Symfony code